barretenberg/cpp/src/barretenberg/vm2/avm_sim_api.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <tuple>
+
+#include "barretenberg/vm2/common/avm_inputs.hpp"
+#include "barretenberg/vm2/proving_helper.hpp"
+#include "barretenberg/vm2/simulation/interfaces/db.hpp"
+#include "barretenberg/world_state/types.hpp"
```

Comment:
these 3 includes and `<tuple>` dont seem needed here. If you use them in the cpp move them there.